sdk/testcontainers-vault/src/options.ts

```diff
@@ -31,6 +31,10 @@ export type VaultOptions = Partial<{
    * The volumes to mount in the Vault container.
    */
   bindMounts: BindMount[];
+  /**
+   * Specify to use Pvault-Server.
+   */
+  isPvaultServer?: Boolean
```

Comment:
- use `boolean`, not `Boolean`.
- rename to `withDB`

```suggestion
  /**
   * Start Vault with a builtin database (default: true).
   */
  withDB?: boolean
```